tests/toMatchScreenshotJest.test.js

```diff
@@ -1,76 +1,124 @@
 const execa = require('execa');
 const dargs = require('dargs');
+const conditionalTest = require('./conditionalTest');
 
 describe('EYES', () => {
   jest.setTimeout(30000);
 
-  describe('Log', () => {
-    test('should log after eyes success', async () => {
-      const options = {
-        config: require.resolve('./__fixtures__/jest-default/conf.json'),
-      };
+  const eyesApiKey = process.env.EYES_API_KEY;
+  describe('when EYES_API_KEY is missing', () => {
+    beforeAll(() => {
+      process.env.EYES_API_KEY = '';
+    });
 
-      const res = await execa('node', [
-        require.resolve('jest/bin/jest'),
-        require.resolve('./__fixtures__/jest-default/test.jest'),
-        dargs(options),
-      ]);
+    test('should return success without accessing eyes', async () => {
+      await global.page.setContent('<div>Hello World 123</div>');
+      const screenshot2 = await global.page.screenshot({fullpage: true});
+      let error;
+      try {
+        await expect(screenshot2).toMatchScreenshot({
+          key: 'Failing Hello World',
+        });
+      } catch (e) {
+        error = e;
+      }
+      expect(error).toBeUndefined();
+    });
 
-      expect(res.stdout).toContain(
-        'eyes comparison succeed for test "should work with jest v1.0.0"',
-      );
+    afterAll(() => {
+      process.env.EYES_API_KEY = eyesApiKey;
     });
   });
 
-  describe('should works with configurations', () => {
-    test('should log after eyes success', async () => {
-      const options = {
-        config: require.resolve('./__fixtures__/jest-with-config/conf.json'),
-      };
+  if (!process.env.EYES_API_KEY) {
+    console.log(
+      '\x1b[33m%s\x1b[0m',
+      '>>> EYES_API_KEY is missing. For all the tests to run please add .env file to the root of the project with EYES_API_KEY=<your key here> <<<',
+    );
```

Comment:
nit: let's move this to the top of the file, so everyone will see it right away when they are writing tests...